tests/eko/evolution_operator/test_ome.py

```diff
@@ -41,7 +41,7 @@ def test_build_ome_as():
         aS = A_singlet((o, 0), N, sx_singlet, nf, L, is_msbar, sx_ns)
 
         for a in [aNS, aS]:
-            for method in ["", "expanded", "exact"]:
+            for method in ["", InversionMethod.EXPANDED, InversionMethod.EXACT]:
```

Comment:
Even the iteration just works:
```suggestion
            for method in InversionMethod:
```